app/_src/gateway/production/monitoring/index.md

```diff
@@ -21,7 +21,23 @@ multiple solutions:
    UDP or TCP and sends aggregated values to one or more backend services. {{site.base_gateway}} directly supports StatsD
    with the [StatsD plugin](/hub/kong-inc/statsd/). [Monitoring with StatsD](/gateway/latest/production/monitoring/statsd/) provides a 
    step-by-step guide to enabling StatsD.
-* For monitoring when {{site.base_gateway}} is ready to accept requests, use the built-in [Node Readiness endpoint](/gateway/latest/production/monitoring/readiness-check/).
+* Use the built-in [Node Readiness endpoint](/gateway/latest/production/monitoring/healthcheck-probes/) for monitoring when {{site.base_gateway}} is ready to accept requests.
 
 Closely related to monitoring is tracing. See the {{site.base_gateway}} [Tracing Reference](/gateway/latest/production/tracing/)
 for details about instrumenting your API gateway.
+
+# What is health checking
+
+Health checking is an activity performed by infrastructure components (i.e. load balancers) to determine the health of a {{site.base_gateway}} node. This helps determine if a {{site.base_gateway}} node is operational and ready to process incoming requests. These health checks are also referred to as "probes" and are explained further in the [Readiness Check](/gateway/latest/production/monitoring/healthcheck-probes/) page.
```

Comment:
```suggestion
Health checking is an activity performed by infrastructure components (that is, load balancers) to monitor the health of a {{site.base_gateway}} node.
This helps determine if a {{site.base_gateway}} node is operational and ready to process incoming requests. 
You can learn more about health checks (also known as "probes") in the [Readiness Check](/gateway/latest/production/monitoring/healthcheck-probes/) guide.
```
Is "i.e" supposed to mean "that is" here, or are load balancers just one example?